### PR TITLE
line number alignment in the search result window

### PR DESCRIPTION
**Pull request by Rudo79**
added appropriate number of leading spaces for line number in the search result window
the issue #11135

### CODE REVIEW

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp [hunk 1]
```diff
@@ -3993,6 +3994,20 @@ void Finder::addFileNameTitle(const TCHAR * fileName)
 	_pMainMarkings->push_back(EmptySearchResultMarking);
 }
 
+void Finder::setLineNumFormatStr()
+{
+   size_t NumLines = (*_ppEditView)->execute(SCI_GETLINECOUNT);
+	size_t NumDigits = 0;
```
Comment (dfs-) [inline PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp:4000]: I guess this shouldn't be indented this far?

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp [hunk 2]
```diff
@@ -3993,6 +3994,20 @@ void Finder::addFileNameTitle(const TCHAR * fileName)
...
+   size_t NumLines = (*_ppEditView)->execute(SCI_GETLINECOUNT);
+	size_t NumDigits = 0;
+   while (NumLines != 0) {
+      NumLines = NumLines / 10;
+      ++NumDigits;
+   }
+
+   TCHAR strFormat[10];
+   wsprintf(strFormat, TEXT("%%%uu"), NumDigits);
+	_LineNumFormatStr = strFormat;
```
Comment (dfs-) [inline PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp:4008]: I guess this shouldn't be indented this far?

**Comment by dfs-** [comment]
Works well for a single file, but looks a bit wonky using Find in Files:

image

Maybe you could modify the code to use the maximum length format for Find in Files?

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp (hunk 1 above)
Comment (Rudo79) [reply PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp:4000]: yes, you are right; I'll correct that together with the improvement for "Find in Files"

**Comment by Rudo79** [comment]
I fixed the coding style of my changes

**Comment by Rudo79** [comment]
I took a look at that it will be little bit more work to implement that.
I will do it in a separate task (if this implementation will be accepted)

**Comment by alankilborn** [comment]
>"Find in Files": I took a look at that it will be little bit more work to implement that.
I will do it in a separate task (if this implementation will be accepted)

I don't think that will be accepted -- solution for all cases should be one submission.

**Comment by Rudo79** [comment]
I'm going to create a new PR with fixed leading spaces for the "Find in Files" as well

**Comment by gawkla** [comment]
Hi all. It should be also possibility to have no preffix in result window.

Other issue is, that search should behave like grep command: if searched sentence exists twice or more in one line this line should be presented once in results. Now each occurence provide one line in output, which easy give double line results, triple line results etc...

**Pull request closed without merging**